source/use_case_ndvi_monitor.rst

```diff
@@ -0,0 +1,151 @@
+   execution of the Use Case under normal, expected conditions. This dialog sequence will ultimately
+   lead to accomplishing the goal stated in the Use Case name and description.
+
+* The user accesses a web page containing information about the current and present vegetation state
+* The user interprets the vegetation state in an extended region (relevant for herding) based on color codes and historic information (using, e.g., a slider)
+
+Alternative Flows
+=================
+
+.. Other, legitimate usage scenarios that can take place within this Use Case.
+
+* The user accesses a mobile app containing information about the current and present vegetation state
+* The user interprets the vegetation state in an extended region (relevant for herding) based on color codes and historic information (using, e.g., a slider)
+
+Exceptions
+==========
+
+.. Anticipated error conditions that could occur during execution of the Use Case, and how the
+   system is to respond to those conditions, or the Use Case execution fails for some reason.
+
+* OPeNDAP access could fail (due to, e.g., high server load?)
+
+Includes
+========
+
+.. Other Use Cases that are included (“called”) by this Use Case (common functionality appearing in
+   multiple Use Cases can be described in a separate Use Case included by the ones that need that
+   common functionality).
+
+* `Investigation of vegetation health variations in a given area based on measurements from Sentinel-2 <ndvi_use_case_investigate>`_
+
+Notes and Issues
+================
+
+.. Additional comments about this Use Case and any remaining open issues that must be resolved. (It
+   is useful to Identify who will resolve each such issue and by what date.)
+
+* We need to make a simple web page to display the results
+* We may want to demonstrate a mobile app
+
+Relevant software
+=================
+
+* Python netCDF4 to access the OPeNDAP stream
+* `Nansat <https://github.com/nansencenter/nansat>`_ could be used as basis for the NDVI tool and to
+  access the Sentinel-2 data stream or any downloaded netCDF-CF files
+
+Relevant datasets
+=================
+
+* NBS Sentinel-2
+
+UML diagram
+===========
+
+.. UML diagram, example;
+
+UML diagram, example;
+
+.. uml::
+
+   @startuml
+   !includeurl https://raw.githubusercontent.com/RicardoNiepel/C4-PlantUML/release/1-0/C4_Context.puml
+
+   LAYOUT_LEFT_RIGHT
+
+   Boundary(producers, "Producers") {
+      System_Ext(ndvi_tool, "NDVI Tool")
+   }
+
+   Boundary(users, "Users") {
+      Person(general_user, "General user")
+   }
+
+   System_Boundary(portals, "Portals") {
+      System_Ext(ndvi_web_app, "NDVI web app")
+      System_Ext(ndvi_mobile_app, "NDVI mobile app")
+   }
+
+   System(senda_find, "S-ENDA Find")
+   System(senda_access, "S-ENDA Access")
+
+   Rel(general_user, portals, "Zooms into region of interest", "Web UI / Mobile app")
+   Rel(ndvi_tool, senda_find, "Registers NDVI dataset", "API")
```

Comment:
Should `ndvi_tool` actually be `producers`?

Reply:
I think so, because this would be an automatic procedure.. @mpejcoch what do you think?

Reply:
I think I misunderstood... So there should be some other tool to do the registration - but this can be "hidden" by just connecting the `producers` box?

Reply:
see 479a2eff2221636596946c0a76e65ca32e001610